AppCenterAnalytics/AppCenterAnalytics/Internals/Session/MSACSessionTracker.h

```diff
@@ -46,6 +46,16 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (void)stop;
 
+/**
+ * Manual session tracking checking.
+ */
+- (void)enableManualSessionTracker;
+
+/**
+ * Start manual session tracking.
```

Comment:
`Start a new session if manual session tracker is enabled`

Let's try to be consistent with other platforms, when writing comments to methods.